directives/btn-group/btn-group.component.js

```diff
@@ -9,7 +9,7 @@
         bindings: {
             items: '<',
             defaultSelected: '<',
-            output: '=',
+            output: '<',
```

Comment:
Use `&` for `output`?

For `defaultSelected`, just call it `selected`? Because in some scenarios, the selected might be changed more than once.

Reply:
again, im using the pattern from here: http://blog.krawaller.se/posts/dissecting-bindings-in-angularjs/

sure

Reply:
Just read it. Fine with this now :)